lib/quantum/job_broadcaster.ex

```diff
@@ -144,6 +173,15 @@ defmodule Quantum.JobBroadcaster do
             "[#{inspect(Node.self())}][#{__MODULE__}] Replacing job #{inspect(job_name)}"
           end)
 
+        # Send event to telemetry incase the end user wants to monitor events
+        :telemetry.execute([:quantum, :job, :add], %{}, %{
```

Comment:
We‘re overriding an existing job here:

```suggestion
        :telemetry.execute([:quantum, :job, :update], %{}, %{
```